app/main.py

```diff
@@ -16,4 +16,28 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
```

Comment:
just populate the dictionary without writing an algorithm

Reply:
corrected